apps/etterlatte-trygdetid-kafka/src/main/kotlin/TrygdetidService.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.etterlatte.trygdetid.kafka
+
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.request.get
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import kotlinx.coroutines.runBlocking
+import no.nav.etterlatte.libs.common.trygdetid.TrygdetidDto
+import java.util.*
+
+class TrygdetidService(
+    private val trygdetidApp: HttpClient,
+    private val url: String
+) {
+
+    fun beregnTrygdetid(behandlingId: UUID): TrygdetidDto = runBlocking {
+        trygdetidApp.get("$url/api/trygdetid/$behandlingId") {
```

Comment:
Litt usikker på hva du regner som scope for denne PR-en, men her vil du vel alltid få `NoContent` tilbake, også vil appen tryne?

Vi er vel nødt til å gjøre en post for å opprette en (tom) trygdetid for behandlingen først, også må vi finne en måte å dytte inn et tall som vi får fra pesys?

Reply:
Øhm, litt usikker på korfor det vart get-kall her, hadde tenkt det skulle vera post 🙈 

Hadde tenkt at scopet for denne er å få ut noko som fungerer, men ikkje nødvendigvis blir rett.